tests/Doctrine/Tests/Common/Cache/ApcuCacheTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Doctrine\Tests\Common\Cache;
+
+use Doctrine\Common\Cache\ApcuCache;
+
+/**
+ * @requires extension apcu
```

Comment:
It is not necessary as it works. If we add support for apcu_entry later we can skip only this specific feature.